File: common/src/main/java/org/ocularscada/devtools/common/TagScripts.java
package org.ocularscada.devtools.common;

import com.inductiveautomation.ignition.common.script.hints.ScriptFunction;

public class TagScripts {

    @ScriptFunction(docBundlePrefix="TagScripts")
    public String helloWorld(String name) { return "Hello " + name; }


}

File: common/src/main/kotlin/org/ocularscada/devtools/common/TagScripts.kt
package org.ocularscada.devtools.common

import com.inductiveautomation.ignition.common.script.hints.ScriptFunction

class TagScripts {

    @ScriptFunction(docBundlePrefix="TagScripts")
    fun helloWorld(name: String): String {
        return "Hello there $name"
    }


}
